src/common/generator.js

```diff
@@ -21,9 +21,21 @@ window.POG=(function() {
             else {
                 selector += '[' + name + '=\'' + value + '\']';
             }
-            if (document.querySelectorAll(selector).length === 1) {
-                response = selector;
-            }
+	    //Surrounding with try to catch the exceptions that occur and crash the program on
+	    //certain web pages. 
+	    try
+	    {
+		if (document.querySelectorAll(selector).length === 1) {
+                    response = selector;
+		}
```

Comment:
@VaccinalBowl I was thinking it would be nice to throw errors directly at the user using:
https://github.com/rickypc/selenium-page-object-generator/blob/master/src/chrome/assets/js/notify.js